Error while compiling openmm-plumed

Hi Team, 
I am new to this type of installations.
I did followed the instructions to compile and got an error while compilation on Mac (High Seirra).
Please see below the error for more details:

_Scanning dependencies of target TestReferencePlumedForce
[ 57%] Building CXX object platforms/reference/tests/CMakeFiles/TestReferencePlumedForce.dir/TestReferencePlumedForce.cpp.o
/Users/nyuad/openmm-plumed/platforms/reference/tests/TestReferencePlumedForce.cpp:83:5: error: expected variable name or 'this' in lambda capture list
    ASSERT_EQUAL_VEC(Vec3(), state.getForces()[1], 1e-5);
    ^
/usr/local/openmm/include/openmm/internal/AssertionUtilities.h:57:174: note: expanded from macro 'ASSERT_EQUAL_VEC'
  ...found, tol) {double _norm_ = std::sqrt((expected).dot(expected)); double _scale_ = _norm_ > 1.0 ? _norm_ : 1.0; if ((std::abs(((expected)[0])-((found)[0]))/_scale_ > (...
                                                                                                                                               ^
/Users/nyuad/openmm-plumed/platforms/reference/tests/TestReferencePlumedForce.cpp:83:5: error: expected variable name or 'this' in lambda capture list
/usr/local/openmm/include/openmm/internal/AssertionUtilities.h:57:234: note: expanded from macro 'ASSERT_EQUAL_VEC'
  ...double _scale_ = _norm_ > 1.0 ? _norm_ : 1.0; if ((std::abs(((expected)[0])-((found)[0]))/_scale_ > (tol)) || (std::abs(((expected)[1])-((found)[1]))/_scale_ > (tol)) ...
                                                                                                                                         ^
/Users/nyuad/openmm-plumed/platforms/reference/tests/TestReferencePlumedForce.cpp:83:5: error: expected variable name or 'this' in lambda capture list
/usr/local/openmm/include/openmm/internal/AssertionUtilities.h:57:294: note: expanded from macro 'ASSERT_EQUAL_VEC'
  ...: 1.0; if ((std::abs(((expected)[0])-((found)[0]))/_scale_ > (tol)) || (std::abs(((expected)[1])-((found)[1]))/_scale_ > (tol)) || (std::abs(((expected)[2])-((found)[2...
                                                                                                                                                              ^
/Users/nyuad/openmm-plumed/platforms/reference/tests/TestReferencePlumedForce.cpp:256:2: error: expected '}'
}
 ^
/Users/nyuad/openmm-plumed/platforms/reference/tests/TestReferencePlumedForce.cpp:83:5: note: to match this '{'
    ASSERT_EQUAL_VEC(Vec3(), state.getForces()[1], 1e-5);
    ^
/usr/local/openmm/include/openmm/internal/AssertionUtilities.h:57:48: note: expanded from macro 'ASSERT_EQUAL_VEC'
#define ASSERT_EQUAL_VEC(expected, found, tol) {double _norm_ = std::sqrt((expected).dot(expected)); double _scale_ = _norm_ > 1.0 ? _norm_ : 1.0; if ((std::abs(((expe...
                                               ^
/Users/nyuad/openmm-plumed/platforms/reference/tests/TestReferencePlumedForce.cpp:256:2: error: expected '}'
}
 ^
/Users/nyuad/openmm-plumed/platforms/reference/tests/TestReferencePlumedForce.cpp:56:18: note: to match this '{'
void testForce() {
                 ^
5 errors generated.
make[2]: *** [platforms/reference/tests/CMakeFiles/TestReferencePlumedForce.dir/TestReferencePlumedForce.cpp.o] Error 1
make[1]: *** [platforms/reference/tests/CMakeFiles/TestReferencePlumedForce.dir/all] Error 2
make: *** [all] Error 2_


Below the cmake output :
CMAKE_BUILD_TYPE                                                                                                                                                             
 CMAKE_EXECUTABLE_FORMAT          MACHO                                                                                                                                       
 CMAKE_INSTALL_PREFIX             /usr/local/openmm                                                                                                                           
 CMAKE_OSX_ARCHITECTURES                                                                                                                                                      
 CMAKE_OSX_DEPLOYMENT_TARGET                                                                                                                                                  
 CMAKE_OSX_SYSROOT                                                                                                                                                            
 CUDA_HOST_COMPILER               /Library/Developer/CommandLineTools/usr/bin/clang                                                                                           
 CUDA_SDK_ROOT_DIR                CUDA_SDK_ROOT_DIR-NOTFOUND                                                                                                                  
 OPENCL_INCLUDE_DIR               /System/Library/Frameworks/OpenCL.framework/Headers                                                                                         
 OPENCL_LIBRARY                   /System/Library/Frameworks/OpenCL.framework                                                                                                 
 OPENMM_DIR                       /usr/local/openmm                                                                                                                           
 PLUMED_BUILD_CUDA_LIB            OFF                                                                                                                                         
 PLUMED_BUILD_OPENCL_LIB          ON                                                                                                                                          
 PLUMED_BUILD_PYTHON_WRAPPERS     ON                                                                                                                                          
 PLUMED_INCLUDE_DIR               /usr/local/include/plumed                                                                                                                   
 PLUMED_LIBRARY_DIR               /usr/local/lib                                                                                                                              
 PYTHON_EXECUTABLE                /Users/nyuad/miniconda3/bin/python                                                                                                          
 SWIG_EXECUTABLE                  /usr/local/bin/swig                                                                                                                         

Can you please advise how to fix this issue? many thanks in advance.

Thanks,
Fahed

--- a/platforms/cuda/tests/TestCudaPlumedForce.cpp
+++ b/platforms/cuda/tests/TestCudaPlumedForce.cpp
@@ -78,11 +78,12 @@ void testForce() {
     State state = context.getState(State::Energy | State::Forces);
     Vec3 delta = positions[0]-positions[2];
     double dist = sqrt(delta.dot(delta));
+    Vec3 zero;
     ASSERT_EQUAL_TOL(dist, state.getPotentialEnergy(), 1e-5);
     ASSERT_EQUAL_VEC(-delta/dist, state.getForces()[0], 1e-5);
-    ASSERT_EQUAL_VEC(Vec3(), state.getForces()[1], 1e-5);
+    ASSERT_EQUAL_VEC(zero, state.getForces()[1], 1e-5);
     ASSERT_EQUAL_VEC(delta/dist, state.getForces()[2], 1e-5);
-    ASSERT_EQUAL_VEC(Vec3(), state.getForces()[3], 1e-5);
+    ASSERT_EQUAL_VEC(zero, state.getForces()[3], 1e-5);
 }
 
 void testMetadynamics() {

--- a/platforms/opencl/tests/TestOpenCLPlumedForce.cpp
+++ b/platforms/opencl/tests/TestOpenCLPlumedForce.cpp
@@ -78,11 +78,12 @@ void testForce() {
     State state = context.getState(State::Energy | State::Forces);
     Vec3 delta = positions[0]-positions[2];
     double dist = sqrt(delta.dot(delta));
+    Vec3 zero;
     ASSERT_EQUAL_TOL(dist, state.getPotentialEnergy(), 1e-5);
     ASSERT_EQUAL_VEC(-delta/dist, state.getForces()[0], 1e-5);
-    ASSERT_EQUAL_VEC(Vec3(), state.getForces()[1], 1e-5);
+    ASSERT_EQUAL_VEC(zero, state.getForces()[1], 1e-5);
     ASSERT_EQUAL_VEC(delta/dist, state.getForces()[2], 1e-5);
-    ASSERT_EQUAL_VEC(Vec3(), state.getForces()[3], 1e-5);
+    ASSERT_EQUAL_VEC(zero, state.getForces()[3], 1e-5);
 }
 
 void testMetadynamics() {

--- a/platforms/reference/tests/TestReferencePlumedForce.cpp
+++ b/platforms/reference/tests/TestReferencePlumedForce.cpp
@@ -78,11 +78,12 @@ void testForce() {
     State state = context.getState(State::Energy | State::Forces);
     Vec3 delta = positions[0]-positions[2];
     double dist = sqrt(delta.dot(delta));
+    Vec3 zero;
     ASSERT_EQUAL_TOL(dist, state.getPotentialEnergy(), 1e-5);
     ASSERT_EQUAL_VEC(-delta/dist, state.getForces()[0], 1e-5);
-    ASSERT_EQUAL_VEC(Vec3(), state.getForces()[1], 1e-5);
+    ASSERT_EQUAL_VEC(zero, state.getForces()[1], 1e-5);
     ASSERT_EQUAL_VEC(delta/dist, state.getForces()[2], 1e-5);
-    ASSERT_EQUAL_VEC(Vec3(), state.getForces()[3], 1e-5);
+    ASSERT_EQUAL_VEC(zero, state.getForces()[3], 1e-5);
 }
 
 void testMetadynamics() {

